File: FMAplication/ClientApp/src/app/Shared/Entity/Sales/salespointStock.ts
import { SalesPoint } from '.';
import { PosmProduct } from '..';
import {Selectable} from '../Common'
export class SalespointStock extends Selectable{
    id: number;
    salesPointId: number;
    salesPoint: SalesPoint;
    posmProductId: number;
    posmProduct: PosmProduct;
    quantity: number;
    availableQuantity:number;
}

export class SalesPointStockTableData{
    id: number;
    salesPointName: string;
    posmProductName: string;
    posmCode:string;
    quantity: number;
    availableQuantity:number;
}

       

File: FMAplication/Domain/Sales/SalesPointStock.cs
using FMAplication.Core;
using FMAplication.Domain.Products;

namespace FMAplication.Domain.Sales
{
    public class SalesPointStock: AuditableEntity<int>
    {
        public int Id { get; set; }
        public int SalesPointId { get; set; }
        public int POSMProductId { get; set; }        
        public POSMProduct POSMProduct { get; set; }
        public int Quantity { get; set; }
    }
}
